### PR TITLE
Fix halo offshoot

### PR DESCRIPTION
**Pull request by m-novikov**

### CODE REVIEW

File: tests/test_lazyflow/test_operators/test_tiktorch/test_classifier.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+def test_get_halo_returns_values_specified_by_tags(pb_session):
+    model_session = ModelSession(session=pb_session, factory=mock.Mock())
+    assert model_session.get_halo(axes="xy") == [256, 128]
+    assert model_session.get_halo(axes="yx") == [128, 256]
+    assert model_session.get_halo(axes="yxc") == [128, 256, 1]
+
+
+def test_get_halo_returns_0_if_value_is_unspecified(pb_session):
+    model_session = ModelSession(session=pb_session, factory=mock.Mock())
+    assert model_session.get_halo(axes="xyz") == [256, 128, 0]
```
Comment (k-dominik) [inline tests/test_lazyflow/test_operators/test_tiktorch/test_classifier.py:27]: philosophical question, is the size of the block along a dimension that is not there, if you're asking for that dimensions,  0, or 1?

**Comment by Tomaz-Vieira** [comment]
This sounds like the kind of problem `ndstructs` was created to solve =)
Maybe we should discuss merging some common codepaths later

File: tests/test_lazyflow/test_operators/test_tiktorch/test_classifier.py (hunk 1 above)
Comment (Tomaz-Vieira) [reply tests/test_lazyflow/test_operators/test_tiktorch/test_classifier.py:27]: I would say that an unmentioned dimension should be `1` if we're talking about a shape, but a `0` if we're talking about a halo

Comment (k-dominik) [reply tests/test_lazyflow/test_operators/test_tiktorch/test_classifier.py:27]: unmentioned? then it should not even exist imo.. if I ask for `xy` then I don't need to have the singletons (or sizes) for `z`

Comment (k-dominik) [reply tests/test_lazyflow/test_operators/test_tiktorch/test_classifier.py:27]: in any case I assume that the code relies on the zeros somewhere. And you are of course right that for a halo one want's 0

**Pull request merged**